dll/shellext/stobject/lang/de-DE.rc

```diff
@@ -19,7 +19,7 @@ BEGIN
     IDS_PWR_METER             "Batterieanzeige ö&ffnen"
     IDS_PWR_RUN               "shell32.dll,Control_RunDLL PowerCfg.cpl"
     IDS_PWR_PERCENT_REMAINING "%.2f%% verbleibend"
-    IDS_PWR_CHARGING          "%.2f%% and charging"
+    IDS_PWR_CHARGING          "%.2f%% aufladen"
```

Comment:
This string indicates "the battery is x% charged, and currently charging". You'd want something like "und wird geladen"